code/modules/hydroponics/grown.dm

```diff
@@ -289,6 +289,20 @@
 		qdel(src)
 		return
 
+	if(seed.kitchen_tag == "carpet")
```

Comment:
This is a good way to obtain otherwise obscure and rare construction item, but I do have suggestion for the future on that front: more subtypes of this plant. Bigger issue with carpets tends to be that subtypes that are not orderable in cargo (default-red and blue) are pretty much unobtainable without adminbus and thats sad. Maybe having some subtypes be xenobotany exclusive and some cargo exclusive as good solution for both sides? Just something to think about.

Reply:
There are two ways I was thinking about going about that, but didn't feel right to put under the purview of this PR in particular:
Option A would be as you say, make more subtypes of carpet-plant, which I worry might end up being rather bloated in the long run, even if making it possible to have a bunch of fun carpets available to the players.

Option B, which I'm more partial to, would be to go the TG station approach. Make Carpet a reagent -- and then add unique recipes to change it further, which replaces tiles by being sprayed upon them. I feel like option B is probably the better approach, and would allow not only xenobotany, but other various disciplines to make unique carpets in the future.